lib/vagrant-service-manager/command.rb

```diff
@@ -17,6 +17,8 @@ module ServiceManager
       'docker' => Docker, 'openshift' => OpenShift,
       'kubernetes' => Kubernetes
     }
+    # Download binary at current folder
+    BIN_FOLDER = "#{ENV['HOME']}/.vagrant.d/data/service-manager/bin"
```

Comment:
ENV['HOME'] is not going to work on Windows in all cases. The Vagrant environment contains the data dir. You should extract it from there